use/DifferentiationSNP.Rmd

```diff
@@ -72,25 +72,25 @@ faster and keep only 20 SNPs in the object `locus`. We then convert `Mydata1` to
 a "hierfstat" object (`Mydata2`).
 
 ```{r,data_conversion}
-
-locus <- Mydata[, 5:24] 
+locus   <- Mydata[, 5:24] 
 Mydata1 <- df2genind(locus, ploidy = 2, ind.names = ind, pop = population, sep = "")
 Mydata1
 Mydata2 <- genind2hierfstat(Mydata1) 
 ```
 
-## $F_{st}$ (observed and expected heterozygosity)  (package hierfstat)
+## $F_{st}$ (observed and expected heterozygosity)
```

Comment:
Is it really a good idea to have parenthetical expressions in headings? How about `Observed and expected heterozygosity: $F_{st}$